challenge_01/data_solution.py

```diff
@@ -0,0 +1,99 @@
+import os
+from typing import List
+
+import torch
+import torch.nn as nn
+import torch.utils.data as data
+from torchvision import transforms
+from torchvision.datasets import CIFAR10
+
+
+class MultiLayerPerceptron(nn.Module):
+    """
```

Comment:
dej si to do souboru models/mlp.py a načítej už (nebo to načítej z base solution)

Reply:
to ma presne napadlo teraz :D